### PR TITLE
Add Benchmarks and CI

### PR DESCRIPTION
**Pull request by bennypowers**
## What I Did

1. Test nullable property access
2. Add a demo of nullable property access
3. Migrate benchmarks to tachometer
4. Add CI via github actions
5. Fix typos
6. Revise README

Supercedes #12

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -119,5 +119,13 @@ <h2>Output:</h2>
       </template>
     </template>
 
+    <template id="demo-8">
+      <template name="title">Null Safe Expressions</template>
+      <template name="demo">
+        <p>This template contains deep access of a nullable object:</p>
+        <div><code>\{{nullable.missing.property || 'none'}}</code>: <span>{{nullable.missing.property || 'none'}}</span></div>
```
Comment (justinfagnani) [inline demo/index.html:126]: oh, we should add the `??` operator to jexpr

File: src/test/stampino_test.ts [hunk 2]
```diff
@@ -159,6 +159,23 @@ suite('lit-html', () => {
     );
   });
 
+  test('nullable model access', function () {
```
Comment (justinfagnani) [inline src/test/stampino_test.ts:162]: This is really more of a jexpr test than a Stampino test. We wouldn't want to test all jexpr features here. Fine for now though.

**Review by justinfagnani — APPROVED** [review]

File: src/test/stampino_test.ts (hunk 2 above)
Comment (bennypowers) [reply src/test/stampino_test.ts:162]: yeah, true, but the (old?) readme mentioned null-safe expressions explicitly, so also for discovery and contributing this is good to have.

File: demo/index.html (hunk 1 above)
Comment (bennypowers) [reply demo/index.html:126]: yes!

**Pull request merged**